superset-frontend/src/SqlLab/components/ResultSet/index.tsx

```diff
@@ -223,7 +248,7 @@ export default class ResultSet extends React.PureComponent<
       // Added compute logic to stop user from being able to Save & Explore
       const { showSaveDatasetModal } = this.state;
       const { query } = this.props;
-
+      console.log('this is query', query);
```

Comment:
remove this console.log

Reply:
@AAfghahi maybe it would be a good idea to get warning on `console` in linter